packages/breadboard/src/runner.ts

```diff
@@ -34,6 +34,14 @@ import { RequestedInputsManager, bubbleUpInputsIfNeeded } from "./bubble.js";
 import { asyncGen } from "./utils/async-gen.js";
 import { StackManager } from "./stack.js";
 import { timestamp } from "./timestamp.js";
+import fs from "fs";
+import path from "path";
+const breadboardSchema = JSON.parse(
+  fs.readFileSync(
+    path.join(process.cwd(), "../schema/breadboard.schema.json"),
```

Comment:
`process.cwd()` is not safe here, because it could be literally anything, since the user can invoke the binary from anywhere in theory. You can use `import.meta.url` to make create paths relative to the current module (https://nodejs.org/api/esm.html#importmetaurl).

But, actually I'm not sure that's what we want here either, because you also can't be sure where on the filesystem the `@google-labs/breadboard-schema` package got installed, since `node_modules/` install layouts can differ quite a bit.

So actually I think we want this?:

```ts
import { createRequire } from "node:module";
const require = createRequire(import.meta.url);
const schemaJson = require("@google-labs/breadboard-schema/breadboard.schema.json");
```

Or if you do need the *path* rather than the *contents*, you can do:

```ts
import { createRequire } from "node:module";
const require = createRequire(import.meta.url);
const schemaPath = require.resolve("@google-labs/breadboard-schema/breadboard.schema.json");
```

Reply:
it needs the change from https://github.com/breadboard-ai/breadboard/pull/581 before I can change it